consultation_analyser/consultations/models.py

```diff
@@ -101,12 +118,33 @@ class Meta:
         ]
 
 
+class AnswerQuerySet(models.QuerySet):
+    MULTIPLE_CHOICE_QUERY = """
```

Comment:
Does this work when multiple options are selected? 
If not, are users prevented from entering multiple options via the schema? (Otherwise it might fail without us realising.)

Reply:
All fine!